services/QuillLMS/client/app/bundles/Teacher/components/accounts/new/onboarding_tour/slide_pagination_button.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+
+const SlidePaginationButton = ({ index, slideIndex, setSlideIndex, }) => {
+  function handleClick() { setSlideIndex(index) }
+
+  const isCurrentSlide = slideIndex === index
+
+  return (
+    <li>
+      <button
+        aria-current={isCurrentSlide ? 'step' : undefined}
+        aria-label={isCurrentSlide ? `Current slide: ${index + 1}` : `Go to slide ${index + 1}`}
```

Comment:
This is super-minor, but it looks like the two text options here are slightly different: the one on the left includes a ":", while the one on the right doesn't.  Given that one is informational and the other is navigational, that may be intentional, but it felt worth flagging in case we want to use the same pattern for both.